src/languages/postgresql.formatter.ts

```diff
@@ -1618,26 +1618,35 @@ const reservedJoins = [
 const reservedDependentClauses = ['WHEN', 'ELSE'];
 
 const binaryOperators = [
+  // Math Operators
   '<<',
   '>>',
-  '||/',
   '|/',
-  '::',
-  ':=',
-  '->>',
-  '->',
-  '#>>',
-  '#>',
-  '=>',
-  '~~*',
+  '||/',
+  '!!',
+  // String Operators
+  '||',
+  // Pattern Matching Operators
   '~~',
-  '!~~*',
+  '~~*',
   '!~~',
+  '!~~*',
+  // POSIX RegExp operators
+  '~',
```

Comment:
Should move `~` to Tokenizer where all other single-letter operators are defined:

```ts
[TokenType.OPERATOR]: {
  regex: regex.operator('+-/*%&|^><=.[]{}`:$@', [
```

Though, looking at this list we should definitely remove the `` ` `` from it, which is never an operator on its own, but instead part of quotes.

Likely should also drop `[]{}` as these are covered by parenthesis patterns. Though perhaps better to leave them for now, as these serve as a fallback in case we have missed to include them to `open/closeParens`. I attempted to [document this syntax](https://github.com/sql-formatter-org/sql-formatter/blob/master/sql/arrays-and-maps.md), but soon found out that I had missed Postgres and BigQuery. Likely there are more. Need to do further investigation.

There are also some omissions in this list of single-letter operators:

- `#` - Postgres bitwise exclusive OR (also a geometric operator)
- `?` - Postgres jsonb operator
- `!` - Don't know of such an operator, but as it can be part of an operator, it's a good fallback to have.